client/src/models/Report.js

```diff
@@ -288,6 +289,17 @@ export default class Report extends Model {
       )
   })
 
+  static autocompleteQuery =
+    "uuid, intent, engagementDate, duration, keyOutcomes, nextSteps, cancelledReason, atmosphere, author { uuid, name, rank, role }, tags { uuid, name, description }, updatedAt"
```

Comment:
Do you really need _all_ these fields? Seeing as the template directly below only uses the intent and the author.

Reply:
Good catch.

Reply:
And `uuid` for row key in the table.